### PR TITLE
change line 6

### PR DESCRIPTION
**Pull request by ramvee321**
resolves # 4

### CODE REVIEW

File: _posts/0000-01-20-ramvee321.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+I wish i would go out
```
Comment (ericalloyd) [inline _posts/0000-01-20-ramvee321.md]: How about we capitalize the "i"?

Comment (ericalloyd) [inline _posts/0000-01-20-ramvee321.md]: Comment

Comment (ericalloyd) [inline _posts/0000-01-20-ramvee321.md]:
```suggestion
I wish I would go out 
```

**Review by ericalloyd — CHANGES_REQUESTED** [review]
LGTM
👍

**Review by ericalloyd — APPROVED** [review]

**Review by dbucke — APPROVED** [review]
looks good

**Pull request merged**